va/validation-authority.go

```diff
@@ -126,16 +126,24 @@ func (va ValidationAuthorityImpl) validateSimpleHTTP(identifier core.AcmeIdentif
 	}
 
 	httpRequest.Host = hostName
-	tr := &http.Transport{
-		// We are talking to a client that does not yet have a certificate,
-		// so we accept a temporary, invalid one.
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-		// We don't expect to make multiple requests to a client, so close
-		// connection immediately.
-		DisableKeepAlives: true,
+	tr := *http.DefaultTransport.(*http.Transport)
```

Comment:
I started with http.DefaultTransport here (instead of initializing a new http.Transport as before) in order to get the usual defaults: 30 second dial timeout and 10 second TLS timeout (instead of unlimited for both), and use of system proxy settings. (The "no timeout, no proxy" situation looked like an accident/bug -- let me know if it was actually a feature.)